io.github.mrquantumoff.mcmodpackmanager.yml

```diff
@@ -0,0 +1,42 @@
+# Flatpak Manifest
+app-id: io.github.mrquantumoff.mcmodpackmanager
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk  
+command: minecraft-modpack-manager
+branch: "master"
+
+modules:
+  - name: mcmodpackmanager
+    buildsystem: simple
+    build-commands:
+      - ar -vox minecraft-modpack-manager_2.4.3_amd64.deb
+      - tar -xpvf data.tar.gz
+      - install -Dm755 -t /app/bin usr/bin/minecraft-modpack-manager
+      - cat desktopfile.desktop
+      - install -D io.github.mrquantumoff.mcmodpackmanager.metainfo.xml /app/share/metainfo/io.github.mrquantumoff.mcmodpackmanager.metainfo.xml
+      - install -D desktopfile.desktop  /app/share/applications/io.github.mrquantumoff.mcmodpackmanager.desktop 
+      - install -D usr/share/icons/hicolor/500x500/apps/minecraft-modpack-manager.png /app/share/icons/hicolor/500x500/apps/io.github.mrquantumoff.mcmodpackmanager.png
+      - install -D usr/share/icons/hicolor/256x256/apps/io.mrquantumoff.mcmodpackmanager.png  /app/share/icons/hicolor/256x256/apps/io.github.mrquantumoff.mcmodpackmanager.png 
+
+    sources:
+      - type: file
+        path: desktopfile.desktop
+      - type: file
+        path: io.github.mrquantumoff.mcmodpackmanager.metainfo.xml
+
+      - type: file
+        url: https://github.com/mrquantumoff/MinecraftModpackManager/releases/download/v2.4.3/minecraft-modpack-manager_2.4.3_amd64.deb
+        sha256: 8c0eb6176ba607589429cc42f43dd1a26a126395235a85e55b4fc6f089897c62
+
+    cleanup:
+      - '*.tar.gz'
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --share=network
+  - --device=dri
+  - --socket=system-bus
```

Comment:
no system bus allowed. You need to figure out which dbus name you need to access.